src/components/Home/Banner/style.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
```

Comment:
here can be used `import { styled } from '@material-ui/core/styles';`

Reply:
if we do this, styled.h1 gives error`Property 'h1' does not exist on type '<Component extends ElementType<any>>(Component: Component) => ComponentCreator<Component>'`